src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -136,7 +136,7 @@
   },
   "stsjorbsmod:Expose": {
     "NAME": "Expose",
-    "DESCRIPTION": "Deal !D! damage. NL If there are enemies still alive, lose !M! HP."
+    "DESCRIPTION": "Deal !D! damage. NL If this does not end the fight, lose !M! HP."
```

Comment:
I like this wording, except maybe use `combat` rather than `fight` for consistency with other card wordings